includes/functions.php

```diff
@@ -550,3 +550,46 @@ function webmention_comment_form() {
 		load_template( $template );
 	}
 }
+
+/**
+ * Refresh an existing comment
+ *
+ * @param int|WP_Comment Comment object or ID.
+ * @return WP_Error|bool Return true or error object.
```

Comment:
isn't it easier to only return "WP_Error" or bool?

Reply:
Isn't that what we're returning?

Reply:
There are a lot of `false` returns.

Reply:
I mean either only WP_Error or only false, but not mixing it.